kimchi/src/verifier.rs

```diff
@@ -63,6 +63,7 @@ impl<G: CommitmentCurve> ProverProof<G>
 where
     G::BaseField: PrimeField,
 {
+    /// ?
```

Comment:
Stop doing this.

Reply:
it's a WIP PR, I can do whatever I want :P

Reply:
You did ask me to look at it!